packages/manager/apps/pci-private-network/src/utils/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { VLAN_ID } from '@/pages/new/new.constants';
+
+const maxRandomNumber = 9999;
+
+const getDate = () => {
+  const date = new Date();
+  return `${date.getDate()}${date.getMonth() + 1}`;
+};
+
+const getRandomNumber = () =>
+  (Math.floor(Math.random() * maxRandomNumber) * new Date().getMilliseconds()) %
+  maxRandomNumber;
+
+export const getAutoGeneratedName = (prefix = ''): string => {
+  const date = getDate();
+  const randomNumber = getRandomNumber();
```

Comment:
Question: is the usage of random number part of the specification?

I mean, it's generally not recommended to use random elements as this, because of performance and side effects  issues.

Maybe the date itself could be sufficient no?

Reply:
I agree I'll check the spec and see if I can use uuid or just a timestamp instead.